src/main/java/ui/components/KeyboardShortcuts.java

```diff
@@ -123,7 +126,7 @@ private static void addNonCustomizableShortcutKeys() {
         assignedKeys.add(KeyCode.F5); //REFRESH
         assignedKeys.add(KeyCode.F1); //SHOW_DOCS
         assignedKeys.add(KeyCode.G); //GOTO_MODIFIER
-        assignedKeys.add(KeyCode.C); //NEW_COMMENT
+        assignedKeys.add(KeyCode.R); //NEW_COMMENT
         assignedKeys.add(KeyCode.A); //MANAGE_ASSIGNEES
         assignedKeys.add(KeyCode.SPACE); //DOUBLE_PRESS
     }
```

Comment:
You could make use of the constants defined above here, then the comments wouldn't be needed.